js/bartender.js

```diff
@@ -1,86 +1,192 @@
+                return 'Sangria Anyone?';
+                }
+        } else {  // selection includes dry
+            if (customerSelection.includes('full-bodied')) {
+                return 'Sinful Zinfandel';
+                } else {  //selection is red, dry, and light
+                    return 'A Light Finish';
+                    }
+            }
+    } else {  // selection includes white
+        if (customerSelection.includes('sweet')) {
+            if (customerSelection.includes('full-bodied')) {
+                return 'A Softer Sancerne';
+                } else {  // selection is white, sweet, and light
+                    return 'Memories of Reisling';
+                    }
+            } else {  // selection is white and dry
+                if (customerSelection.includes('light')) {
+                      return 'A Petite Pinot Gris';
+                      } else {  //selection is white, dry, and light
+                          return 'Bite Your Tongue';
+                          }
+                    }
+            }
+};
 
-        wineName = bartender.newBlend(Preference.preferences);
+$(document).ready(function() {
 
-        $('.deliver-blend').show();
-        $('.deliver-blend').prepend("<p>" + bartender.name + " created a \"" + wineName + "\" wine blend for you!</p>");
-        console.log(bartender.name + " created a " + wineName + " for you!");
+    $('.barkeep').hide();
+    $('.customer').hide();
 
-      });
+    // check user role
+    $('.user-role').click( function(e) {
+        e.preventDefault();
+        role = $('input[name=serveOrsip]:checked').val();
+        $('.user-role').hide();
+
+        if (role === 'serve') {
+            // stock the wine cellar
+            $('.barkeep').show();
+
+            $('input[type=submit]').click( function(e) {
+                e.preventDefault();
+                var wineType = $('input[name=wineName]').val();
+                $('input[name=wineName]').val('');
+                wineColor = $('input[name=wineColor]:checked').val();
+                wineFlavor = $('input[name=wineFlavor]:checked').val();
+                wineWeight = $('input[name=wineWeight]:checked').val();
+
+                wineInventory.stockWine(wineType, wineColor, wineFlavor, wineWeight);
+
+                $('.restock-done').click( function(e) {
+                    e.preventDefault();
+                    if('input[name=restock]:checked') {
+                        // hide barkeep content, clear user role, clear radio buttons and checkbox
+                        $('.barkeep').hide();
+                        role = '';
+                        $('input[name=wineName]').val('');
+                        $('input[name=wineColor]').prop('checked', false);
+                        $('input[name=wineFlavor]').prop('checked', false);
+                        $('input[name=wineWeight]').prop('checked', false);
+                        $('input[name=restock]').prop('checked', false);
```

Comment:
these selectors are not safe, there's another input element with each of these name attributes in the customer div. The selector will choose both and even though in this particular case there is no harm since only one of the divs is visible at a time, however it is dangerous. I'd rather u assign an id to each and use that instead.